Source/WebCore/rendering/RenderLayerModelObject.h

```diff
@@ -73,6 +75,10 @@ class RenderLayerModelObject : public RenderElement {
     void suspendAnimations(MonotonicTime = MonotonicTime()) override;
 
 #if ENABLE(LAYER_BASED_SVG_ENGINE)
+    // Single source of truth deciding if a SVG renderer should be painted. All SVG renderers
+    // use this method to test if they should continue processing in the paint() function or stop.
+    bool shouldPaintSVGRenderer(const PaintInfo&, const StdUnorderedSet<PaintPhase>& relevantPaintPhases) const;
```

Comment:
Refactoring this separately makes reviewing easier.

Reply:
Ok, will split it up - sorry for the delay.

Reply:
Just splitted it up in a 2nd commit - IMHO that doesn't deserve an extra ticket / PR.